### PR TITLE
[Files] Add and improve Spanish translations

### LINKED ISSUE #2128: [Files] Add and improve Spanish translations
The idea is to add missing translations and improve the ones that are not correct in Spanish language

### PR DESCRIPTION
**Pull request by juans-chainsafe**
closes #2128

### CODE REVIEW

File: packages/files-ui/src/locales/de/messages.po [hunk 1]
```diff
@@ -307,8 +307,8 @@ msgstr ""
 msgid "Device awaiting confirmation"
 msgstr "Gerät wartet auf Bestätigung"
 
-msgid "Didn't receive the email ?"
-msgstr "Sie haben die E-Mail nicht erhalten?"
+msgid "Didn't receive the email?"
+msgstr ""
```
Comment (Tbaut) [inline packages/files-ui/src/locales/de/messages.po]:
```suggestion
msgstr "Sie haben die E-Mail nicht erhalten?"
```

File: packages/files-ui/src/locales/en/messages.po [hunk 2]
```diff
@@ -418,6 +418,9 @@ msgstr "Files Pro"
 msgid "Files sharing key"
 msgstr "Files sharing key"
 
+msgid "Folder Name"
+msgstr "Nombre de la carpeta"
```
Comment (Tbaut) [inline packages/files-ui/src/locales/en/messages.po]: This is the english file.

```suggestion
msgstr "Folder Name"
```

File: packages/files-ui/src/locales/fr/messages.po [hunk 3]
```diff
@@ -308,8 +308,8 @@ msgstr "L'adresse de destination"
 msgid "Device awaiting confirmation"
 msgstr "Appareil en attente de confirmation"
 
-msgid "Didn't receive the email ?"
-msgstr "Vous n’avez pas reçu de courriel ?"
+msgid "Didn't receive the email?"
+msgstr ""
```
Comment (Tbaut) [inline packages/files-ui/src/locales/fr/messages.po]:
```suggestion
msgstr "Vous n’avez pas reçu de courriel ?"
```

File: packages/files-ui/src/locales/no/messages.po [hunk 4]
```diff
@@ -307,8 +307,8 @@ msgstr ""
 msgid "Device awaiting confirmation"
 msgstr ""
 
-msgid "Didn't receive the email ?"
-msgstr "Fikk du ikke e-posten?"
+msgid "Didn't receive the email?"
+msgstr ""
```
Comment (Tbaut) [inline packages/files-ui/src/locales/no/messages.po]:
```suggestion
msgstr "Fikk du ikke e-posten?"
```

**Review by Tbaut — APPROVED** [review]
Huge work! Thanks for spoting all these cases without translation.
I left minor comments and suggestions

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 5]
```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { ToastContent, ToastParams, ToastProvider, useToasts } from "../Toasts"
 import { withKnobs, number, select, text, boolean } from "@storybook/addon-knobs"
 import { action } from "@storybook/addon-actions"
+import { t } from "@lingui/macro"
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]: I realized this is in a `.stories` files. it's not relevant then, these are files for dev to tests the components. We should revert this
```suggestion
```

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 6]
```diff
@@ -14,7 +15,7 @@ export const ToastsContentNotification: React.FC = () => {
     <ToastContent
       toast={{
         id: "1",
-        title: text("title", "Upload in progress"),
+        title: text("title", t`Upload in progress`),
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]:
```suggestion
        title: text("title", "Upload in progress"),
```

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 7]
```diff
@@ -28,7 +29,7 @@ export const ToastsContentProgress: React.FC = () => {
     <ToastContent
       toast={{
         id: "2",
-        title: text("title", "Upload in progress"),
+        title: text("title", t`"Upload in progress`),
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]:
```suggestion
        title: text("title", "Upload in progress"),
```

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 8]
```diff
@@ -43,7 +44,7 @@ export const ToastsContentProgressNoCancel: React.FC = () => {
     <ToastContent
       toast={{
         id: "2",
-        title: text("title", "Upload in progress"),
+        title: text("title", t`Upload in progress`),
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]:
```suggestion
        title: text("title", "Upload in progress"),
```

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 9]
```diff
@@ -60,7 +61,7 @@ export const ToastsDemo: React.FC = () => {
     >
       <ToastNotificationDemo
         toast={{
-          title: text("title", "Upload Complete"),
+          title: text("title", t`Upload Complete`),
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]:
```suggestion
          title: text("title", "Upload Complete"`),
```

File: packages/common-components/src/stories/Toasts.stories.tsx [hunk 10]
```diff
@@ -79,7 +80,7 @@ export const MultipleToastsDemo: React.FC = () => {
     >
       <ToastMultipleNotificationDemo
         toast={{
-          title: text("title", "Upload Started"),
+          title: text("title", t`Upload Started`),
```
Comment (Tbaut) [inline packages/common-components/src/stories/Toasts.stories.tsx]:
```suggestion
          title: text("title", "Upload Started"),
```

**Review by Tbaut — COMMENTED** [review]
I thought that you were playing with the UI and realized that some element hadn't been translated. I figured you must have looked in the code directly, and indeed those stories files do not need to be updated, that's the reason why lingui, that we use for translation, didn't pick them up, and ended up deleting any translation it didn't know.

**Comment by Tbaut** [comment]
BTW I found some other string that hadn't been translated. I pushed it directly

**Comment by juans-chainsafe** [comment]
> BTW I found some other string that hadn't been translated. I pushed it directly

I really appreciate that you found the problem! thanks! I will translate the strings that you found

File: packages/storage-ui/cypress/tests/bucket-management-spec.ts [hunk 11]
```diff
@@ -42,7 +42,6 @@ describe("Bucket management", () => {
       fileUploadModal.uploadButton().safeClick()
       fileUploadModal.body().should("not.exist")
       uploadStatusToast.body().should("be.visible")
-      bucketContentsPage.awaitBucketRefresh()
```
Comment (Tbaut) [inline packages/storage-ui/cypress/tests/bucket-management-spec.ts]: this looks unrelated to this PR. I'd suggest to keep this PR focused on the spanish translation :)

Comment (juans-chainsafe) [reply packages/storage-ui/cypress/tests/bucket-management-spec.ts]: yes! I just did the undo, I was in the wrong branch

**Review by tanmoyAtb — APPROVED** [review]
Looks good to me ! amazing work !
